djangocms_admin_style/sass/components/_base.scss

```diff
@@ -195,6 +195,21 @@ ul.errorlist {
     }
 }
 
+.messagelist {
+    position: absolute;
+    top: 0;
+    left: 3%;
+    right: 3%;
+    margin-top: $messages-margin-top !important;
+    li {
+        color: $color-success !important;
+        font-weight: normal !important;
+        padding: 2px 10px !important;
+        background-image: none !important;
+        border-bottom: 0 !important;
```

Comment:
Do you really need all this importants? Maybe you can use some id or class under the class.